module-audio/Audio/Profiles/ProfileRoutingEarspeaker.hpp

```diff
@@ -21,10 +21,13 @@ namespace audio
                       .flags =
                           static_cast<uint32_t>(audio::codec::Flags::InputLeft) | // microphone use left audio channel
                           static_cast<uint32_t>(audio::codec::Flags::OutputMono),
-                      .outputVolume = 0,
-                      .inputGain    = 0,
-                      .inputPath    = audio::codec::InputPath::Microphone,
-                      .outputPath   = audio::codec::OutputPath::Earspeaker,
+                      .outputVolume      = 0,
+                      .inputGain         = 0,
+                      .playbackPathGain  = 5,
```

Comment:
in other cases plaubackPath gain is set to 0 ( or default to 0) here it's to 5 - it's not a typo? :)

Reply:
It's a typo indeed, should be set to 0 - thanks!